neptune/alpha/internal/init_impl.py

```diff
@@ -99,21 +101,27 @@ def init(
         exp = backend.create_experiment(project_obj.uuid, git_ref)
 
     if connection_mode == "async":
-        experiment_path = "{}/{}".format(NEPTUNE_EXPERIMENT_DIRECTORY, exp.uuid)
+        experiment_path = "{}/{}/{}".format(NEPTUNE_EXPERIMENT_DIRECTORY, ASYNC_DIRECTORY, exp.uuid)
+        try:
+            execution_id = len(os.listdir(experiment_path))
+        except FileNotFoundError:
+            execution_id = 0
+        execution_path = "{}/exec-{}-{}".format(experiment_path, execution_id, datetime.now())
         operation_processor = AsyncOperationProcessor(
             exp.uuid,
-            DiskQueue(experiment_path,
+            DiskQueue(execution_path,
                       OPERATIONS_DISK_QUEUE_PREFIX,
                       VersionedOperation.to_dict,
                       VersionedOperation.from_dict),
             backend,
-            SyncOffsetFile(Path(experiment_path)),
+            SyncOffsetFile(Path(execution_path)),
             sleep_time=flush_period)
     elif connection_mode == "sync":
         operation_processor = SyncOperationProcessor(exp.uuid, backend)
     elif connection_mode == "debug":
         operation_processor = SyncOperationProcessor(exp.uuid, backend)
     elif connection_mode == "offline":
+        # Experiment was returned by mocked backend and has some random UUID.
```

Comment:
Now that we raise `ExperimentNotFound` in mock_backend's get_experiment, this is no longer true, is it?

Reply:
I belive we use `create_experiment` method in happy path in offline mode.